### ISSUE TITLE
Highlight important fact about KV secrets doc

### ISSUE THREAD

**Issue by 76creates** [hashicorp/vault#17425]
I have spent some time debugging my issue and while googling noticed folks are having same questions as I do, its about KV engine documentation and the fact that very important fact:`This documentation assumes the kv secrets engine is enabled at the /secret path` is not highlighted at all. It's very plain value to the point that inexperienced user just wanting to do a quick fetch of API call URL could make a mistake that `/secret` is actual part of the handler. It would be **super useful** to make sure this block of text is highlighted in some color. Thanks ⚡ 

TLDR request:
Please highlight following blocks of text with some color:
https://github.com/hashicorp/vault/blob/418e528008fb7593c1df5c4f17bd0c9406123698/website/content/api-docs/secret/kv/kv-v1.mdx?plain=1#L13-L15
https://github.com/hashicorp/vault/blob/418e528008fb7593c1df5c4f17bd0c9406123698/website/content/api-docs/secret/kv/kv-v1.mdx?plain=1#L13-L15

**Comment by hsimon-hashicorp** [comment]
Hi there! Thanks for this - I'll tag it for our docs team, however, please do feel free to create a PR that closes this issue in the meantime. It seems like a pretty easy change, but with the team just coming back from HashiConf, I don't want to make you wait unnecessarily. :)
